test/e2e/yaml/e2e-tools.yaml

```diff
@@ -55,3 +55,25 @@ tools:
         valuefile: values.yaml
         path: helm/${{github-repo-scaffolding-golang.default.outputs.repo}}
         repoURL: ${{github-repo-scaffolding-golang.default.outputs.repoURL}}
+  - name: trello
+    instanceID: default
+    dependsOn: [ ]
+    options:
+      org: [[ githubOrganization ]]
+      repo: [ [ RepoName ] ]
```

Comment:
The "[ [" and "] ]" should be "[[" and "]]". Same as above and below.

Reply:
has been fixed.